pkg/operator/operator.go

```diff
@@ -49,6 +50,10 @@ var (
 	log = logf.Logger.WithName("init")
 )
 
+const (
```

Comment:
I think it would be more consistent to add the `os.Getenv` to start.go and add this variable as a part of the `operatorconfig.Config` object in config/config.go. That'd at least keeps it aligned with other configuration options. Cluster-dns-operator uses ENV variables exclusively, and it uses the `operatorconfig.Config` pattern: https://github.com/openshift/cluster-dns-operator/blob/master/cmd/dns-operator/main.go#L23-L48 so I'm using that as a guide.